tests/test_core_network.py

```diff
@@ -219,6 +220,20 @@ def network4():
     return n4
 
 
+@pytest.fixture()
+def network5(correct_schedule):
```

Comment:
so I know you're following 'convention' of networkInteger - but it's really not a great convention, let's be better than that 😂 i would call this fixture: `network_for_summary_stats` and also include the expected output in the fixture, so it would return a dictionary: {'network_object': n, 'summary_output': ...} and then your test at the bottom will be ~ three lines